lib/dice_bag/private_key.rb

```diff
@@ -9,32 +8,33 @@ def initialize(key)
       @private_key = key
     end
 
-    def is_valid_private_key?
+    def valid_private_key?
       require 'openssl'
+
       begin
         OpenSSL::PKey::RSA.new @private_key
-        return true  
+        true
       rescue => e
         p e.message
         p e.backtrace
-        return false
+
+        false
       end
-    end 
+    end
 
     def to_rsa_format!
       strip_down_key
       body = @private_key.split(/\s+/)
       body = body.first.scan(/.{1,64}/) if body.length == 1
-    	@private_key = [@@header, body, @@footer].flatten.join("\n")
+      @private_key = [@@header, body, @@footer].flatten.join("\n")
```

Comment:
for another PR too but those nasty class vars should really be constants, unless some hidden case I'm missing here.

Reply:
Yeah I was put off and thought of another PR, but a second look with a grep, seems like the use case is really limited here, all inside the class. So updated to constantize them e909872.